commands/export.go

```diff
@@ -0,0 +1,213 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/mattermost/mmctl/client"
+	"github.com/mattermost/mmctl/printer"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/spf13/cobra"
+)
+
+var ExportCmd = &cobra.Command{
+	Use:   "export",
+	Short: "Management of exports",
+}
+
+var ExportCreateCmd = &cobra.Command{
+	Use:   "create",
+	Short: "Create export file",
+	Args:  cobra.NoArgs,
+	RunE:  withClient(exportCreateCmdF),
+}
+
+var ExportDownloadCmd = &cobra.Command{
+	Use:     "download [exportname] [filepath]",
+	Short:   "Download export files",
+	Example: " export download export_file.zip",
```

Comment:
Can we add a couple of examples following the new format here? I was a bit confused with the command at first. Maybe something like:

```suggestion
	Example: `  # you can indicate the name of the export and it's destination path
  $ mmctl export download samplename sample_export.zip
  
  # or if you only indicate the name, the path would match it
  $ mmctl export download sample_export.zip`,
```